packages/storage-events/src/replay.js

```diff
@@ -42,7 +42,7 @@ function changedTodo(todos, data) {
     } else {
       return {
         ...todo,
-        children: changedTodo(todo.children, data)
+        children: changedTodo(todo.children || [], data)
```

Comment:
Shall we set `todo.children` to `[]`, whenever we create a new todo? Otherwise we need to check if `.children` is set everywhere...